README.md

```diff
@@ -1,107 +1,45 @@
-version: 0.1.0
-depends:
-  - name: vmware
-    url: https://github.com/chris-rock/inspec-vsphere/archive/master.tar.gz
+export VISERVER=10.0.0.10
+export VISERVER_USERNAME=demouser
+export VISERVER_PASSWORD=s0m3t1ngs3cuRe
+inspec exec inspec-powercli -t vmware://
 ```
 
-### Add controls
-
-Since your profile depends on the resource pack, you can use those resources in your own profile:
+### Via the InSpec Shell
 
 ```
-control "vmware-1" do
-  impact 0.7
-  title 'Checks that soft power off is diabled'
-  describe vmware_vm_advancedsetting({datacenter: 'ha-datacenter', vm: 'testvm'}) do
-    its('softPowerOff') { should cmp 'false' }
-  end
-end
+inspec shell -t vmware://USERNAME@VISERVER --password MY_PASSWORD --depends ./inspec-powercli
 ```
 
-### Available Resources
-
- * `vmhost_acceptance` - This resource reads the switch configuration of a hostsystem
- * `vmhost_advancedsetting` - This resource reads all host advanced configuration options.
- * `vmhost_buildnumber` - This resource reads the actual build number of a hostsystem.
- * `vmhost_lockdown` - This resource reads the lockdown mode of a hostsystem.
- * `vmhost_ntp` - This resource reads the ntp configuration of a hostsystem.
- * `vmhost_service` - This resource reads service information of the host system.
- * `vmhost_vswitch` - This resource reads the vswitch configuration of a hostsystem
- * `vm_advancedsetting` - This resource reads all vm advanced configuration options.
- * `vm_device` - This resource reads all vm device configuration options.
- * `virtual_portgroup` - This resource reads the portgroup of a hostsystem.
- * `vmhost_firewall` - This resource reads the firewall configuration of a hostsystem.
-
-### Roadmap
-
- * `vm_harddisk`
- * `vmhost_coredump`
- * `vmhost_account`
- * `vmhost_authentication`
- * `vmhost_webserver`
- * `vmhost_module`
- * `vmhost_vib`
- * `vmhost_iscsi`
-
-## Pre-Requirements
+## Installing PowerCLI
 
+### Windows 2012R2
 ```
-# inspec
-gem install inspec
-# gem for VMware vSphere API
-gem install rbvmomi
+Invoke-WebRequest -Uri "https://download.microsoft.com/download/C/4/1/C41378D4-7F41-4BBE-9D0D-0E4F98585C61/PackageManagement_x64.msi" -OutFile PackageManagement.msi
+msiexec.exe /i C:\Users\vagrant\PackageManagement.msi /quiet
+Set-PSRepository -Name "PSGallery" -InstallationPolicy "Trusted"
+Install-Module -Name VMware.PowerCLI
 ```
 
-# vSphere API Explorer
-
-You can inspect the vSphere API via `/mob` on your ESXi server.
-
-## License
-
-|  |  |
-| ------ | --- |
-| **Author:** | Christoph Hartmann (<chris@lollyrock.com>) |
-| **Copyright:** | Copyright (c) 2017 Chef Software Inc. |
-| **Copyright:** | Copyright (c) 2016 Christoph Hartmann |
-| **License:** | Apache License, Version 2.0 |
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
+### Linux
```

Comment:
We should add the Ubuntu http://jjasghar.github.io/blog/2018/03/22/powercli-10+-on-linux/ installation steps here too, "offically" Ubuntu is the only supported pwsh+PowerCLI platform from VMware ATM.

Reply:
I'll replace with these instructions then